StRoot/StFttDbMaker/StFttDbMaker.cxx

```diff
@@ -0,0 +1,46 @@
+/***************************************************************************
+ * $id: StFttDbMaker.cxx,v 1.0 2020/12/17 21:01:04 jdb Exp $
+ * \author: jdb
+ ***************************************************************************
+ *
+ * Description: This maker is the interface between FCS and the STAR database
```

Comment:
FCS?

Reply:
Addressed in 3686a6eea3b300c95180bdcee0ad698b5763a88c